diff --git a/diode_tht_sod123.js b/diode_tht_sod123.js
index 6f4a602..5a938c8 100644
--- a/diode_tht_sod123.js
+++ b/diode_tht_sod123.js
@@ -128,8 +128,8 @@ module.exports = {
         `
 
     const front_smd_pads = `
-      (pad "1" smd rect (at -1.65 0 ${p.r}) (size 0.9 1.2) (layers "F.Cu" "F.Paste" "F.Mask") ${p.to.str})
-      (pad "2" smd rect (at 1.65 0 ${p.r}) (size 0.9 1.2) (layers "F.Cu" "F.Paste" "F.Mask") ${p.from.str})
+      (pad "1" smd rect (at -1.95 0 ${p.r}) (size 1.2 1.2) (layers "F.Cu" "F.Paste" "F.Mask") ${p.to.str})
+      (pad "2" smd rect (at 1.95 0 ${p.r}) (size 1.2 1.2) (layers "F.Cu" "F.Paste" "F.Mask") ${p.from.str})
         `
     const back_silk = `
       (fp_line (start 0.25 0) (end 0.75 0) (layer "B.SilkS") (stroke (width 0.1) (type solid)))
@@ -141,13 +141,13 @@ module.exports = {
       (fp_line (start -0.75 0) (end -0.35 0) (layer "B.SilkS") (stroke (width 0.1) (type solid)))
         `
     const back_smd_pads = `
-      (pad "1" smd rect (at -1.65 0 ${p.r}) (size 0.9 1.2) (layers "B.Cu" "B.Paste" "B.Mask") ${p.to.str})
-      (pad "2" smd rect (at 1.65 0 ${p.r}) (size 0.9 1.2) (layers "B.Cu" "B.Paste" "B.Mask") ${p.from.str})
+      (pad "1" smd rect (at -1.95 0 ${p.r}) (size 1.2 1.2) (layers "B.Cu" "B.Paste" "B.Mask") ${p.to.str})
+      (pad "2" smd rect (at 1.95 0 ${p.r}) (size 1.2 1.2) (layers "B.Cu" "B.Paste" "B.Mask") ${p.from.str})
         `
       
     const reversible_tht_pads = `
-      (pad "1" thru_hole rect (at -1.65 0 ${p.r}) (size 0.9 1.2) (drill 0.3) (layers "*.Cu" "*.Paste" "*.Mask") ${p.to.str})
-      (pad "2" thru_hole rect (at 1.65 0 ${p.r}) (size 0.9 1.2) (drill 0.3) (layers "*.Cu" "*.Paste" "*.Mask") ${p.from.str})
+      (pad "1" thru_hole rect (at -1.95 0 ${p.r}) (size 1.2 1.2) (drill 0.3) (layers "*.Cu" "*.Paste" "*.Mask") ${p.to.str})
+      (pad "2" thru_hole rect (at 1.95 0 ${p.r}) (size 1.2 1.2) (drill 0.3) (layers "*.Cu" "*.Paste" "*.Mask") ${p.from.str})
         `
 
     const tht = `
